ENIGMAsystem/SHELL/Bridges/General/Win32-Direct3D11.cpp

```diff
@@ -122,105 +122,156 @@ void initialize_render_targets() {
 
 } // namespace anonymous
 
-namespace enigma
-{
-  extern void (*WindowResizedCallback)();
-  void WindowResized() {
+namespace enigma {
 
-    // release all the old references first
-    m_deviceContext->OMSetRenderTargets(0, 0, 0);
-    m_renderTargetView->Release();
-    m_depthStencilBuffer->Release();
-    m_depthStencilView->Release();
+extern void (*WindowResizedCallback)();
+void WindowResized() {
+  // release all the old references first
+  m_deviceContext->OMSetRenderTargets(0, 0, 0);
```

Comment:
This group of lines looks common, enumerative, and easy to neglect. Is there nowhere else in the engine that needs to dispose of stale contexts?

Reply:
No, not yet, and the issue is the whole thing is just... componentized... which is common for more modern graphics APIs as we know. So, you know, a render target can be a texture or a texture plus a depth buffer.